src/main/resources/assets/nochatreports/lang/en_us.json

```
@@ -10,5 +10,40 @@
 	"gui.nochatreports.unsafe_server.check": "Do not show again for this server",
-	"gui.nochatrepords.reload_config_tooltip": "Reload No Chat Reports config. Use carefully."
-}
+	"gui.nochatrepords.reload_config_tooltip": "Reload No Chat Reports config. Use carefully.",
+
+	"configuration.NoChatReports.config": "No Chat Reports",
+	"configuration.NoChatReports.category.visual": "Visual",
+	"configuration.NoChatReports.category.technical": "Technical",
+	"configuration.NoChatReports.category.whitelistedServers": "Whitelisted Servers",
+	"option.NoChatReports.demandOnServer": "Demand on server",
+	"option.NoChatReports.demandOnServer.tooltip": "Prevents the client from joining servers that don't\nhave No Chat Reports installed.",
+	"option.NoChatReports.enableDebugLog": "Enable debug log",
+	"option.NoChatReports.enableDebugLog.tooltip": "Enables logging of certain debug information.\nThis is generally useless for non-developers.",
+	"option.NoChatReports.convertToGameMessage": "Convert to game message",
+	"option.NoChatReports.convertToGameMessage.tooltip": "Makes the server convert all player messages into system\nmessages. System messages don't have any signatures\nattached and cannot be reported; on unmodified clients\nthey will display with gray bar to the left.",
+	"option.NoChatReports.showServerSafety":"Show server safety",
+	"option.NoChatReports.showServerSafety.tooltip":"Allows the client to display server safety status in\nbottom-right corner of chat screen. The icon is\nhoverable for additional information:",
+	"option.NoChatReports.showServerSafety.moreInfo":"More information on server safety levels linked here",
+	"option.NoChatReports.hideRedChatIndicators":"Hide red chat indicators",
+	"option.NoChatReports.hideRedChatIndicators.tooltip":"Prevents the client from displaying \"Not Secure\"\nindicators, in the form of red bar to the left\nand §c§l(§r§c!§c§l)§r icon to the right of chat message.",
+	"option.NoChatReports.hideYellowChatIndicators":"Hide yellow chat indicators",
+	"option.NoChatReports.hideYellowChatIndicators.tooltip":"Prevents the client from displaying \"Modified\"\nindicators, in the form of yellow bar to the left\nand §e§l(§r§e?§e§l)§r icon to the right of chat message.",
+	"option.NoChatReports.hideGrayChatIndicators":"Hide gray chat indicators",
+	"option.NoChatReports.hideGrayChatIndicators.tooltip":"Prevents the client from displaying special\nindicators for system messages, in the form\nof gray bar to the left.",
+	"option.NoChatReports.hideWarningToast":"Hide warning toast",
+	"option.NoChatReports.hideWarningToast.tooltip":"Removes a warning toast that is displayed when entering\nservers with disabled §lenforce-secure-profile§r.",
```

```suggestion
	"option.NoChatReports.hideWarningToast.tooltip":"Removes a warning toast that is displayed when entering\nservers with disabled §oenforce-secure-profile§r.",
```
I would make it italic instead of bold

---

Not resolved

---

Resolved now.